docs/source/release-history.rst

```diff
@@ -7,6 +7,40 @@ In Development
 
 [Add information about changes in your PR here]
 
+**Breaking Changes:**
+
+- The order of parameters in :meth:`wayback.WaybackClient.get_memento` has changed, and additional parameters for the date and mode of memento playback have been added. You can now call ``get_memento()`` with the URL and time you want the memento of, rather than needing to compose a complete Wayback memento URL. That is, you can now call:
+
+  .. code-block:: python
+
+     client.get_memento('http://www.noaa.gov/',
+                        datetime(2018, 8, 16, 11, 19, 11))
```

Comment:
you still need to know something about date formatting in order to know what's going on here, it would be helpful to see a line like "where datetime's arguments are (4-digit year, month number, day of month, ...)"

Reply:
or even e.g. "uses standard UTC" would be a nice breadcrumb

Reply:
Do you really think how to use `datetime` needs to be spelled out here? It’s part of Python’s standard library and widely used all over the place.

Re: UTC, good point, although `datetime` can carry timezone information, and if it does, we respect it. So it’s more that we assume UTC if you don’t specify a timezone. Anyway, I’ll add something to that effect.

FWIW, I don’t necessarily expect anything here to be the final release notes. Since this is going to be a breaking change, I’m going to include #2 in this release, and between this PR and that, we’re pretty much totally redo-ing everything about `get_memento`. So it’s going to need much more explanation that might involve rewriting a bunch of this.

Reply:
no, you're right. This is fine

Reply:
Cool. I will still update to clarify that it gets interpreted as UTC when you don’t explicitly specify a timezone. That’s a good catch! 👍